packages/open-next/src/adapters/cache.ts

```diff
@@ -490,7 +490,7 @@ export default class S3Cache {
       );
       return result;
     } catch (e) {
-      error("This error can usually be ignored : ", e);
+      warn("This error can usually be ignored : ", e);
```

Comment:
How often are you seeing this error? If it's usually ignorable, then should we just use `debug` or omit the log altogether?

Reply:
It seems to be in weird scenarios, like when sustaining attacks with weird things on the end of URLs (\ is common).

All in all, not too much of an issue, I can ignore on our side, but I doubt we are the only website on the internet to get attacked by bots.

Reply:
I also think `warn` is appropriate.  Something weird happened, but no need to get out of bed in the middle of the night (unlike error).

Reply:
Is it a denial of wallet attack? Is the S3 LIST adding up on your account?

Reply:
Seems more like probing for now, but yes, I would assume it is to try and throw a wrench into some database call or something.

Reply:
We should probably modify the `awslogger` that we pass to S3 as well.
https://github.com/sst/open-next/blob/e32af92b30738a52066208b8ea39a71fdb3be67f/packages/open-next/src/adapters/server-adapter.ts#L35

Reply:
> We should probably modify the `awslogger` that we pass to S3 as well.
> 
> https://github.com/sst/open-next/blob/e32af92b30738a52066208b8ea39a71fdb3be67f/packages/open-next/src/adapters/server-adapter.ts#L35

Done.